deploy/crds/operator.ibm.com_v1alpha1_operandconfig_cr.yaml

```diff
@@ -71,3 +71,193 @@ spec:
   - name: ibm-monitoring-grafana-operator
     spec:
       grafana: {}
+  - name: ibm-elastic-stack-operator
+    spec:
+#      elasticStack: {}
+      elasticStack:
```

Comment:
please remove the parameters in this CR.

This OperandConfig CR is managed by administrators and only the administrator faced parameters can be added here.